lib/plugins/http-error-handler.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+function httpErrorHandler(request, reply) {
+  const response = request.response;
+  if (response.isBoom) {
+    let msg;
+    let statusCode = response.output.payload.statusCode;
+    let errorTitle = response.output.payload.error;
+    let errorMessage = response.output.payload.message;
```

Comment:
Use `const` for variables that don't change. Also, you're not using the `msg` variable.